src/settings.lris_red

```diff
@@ -39,13 +39,15 @@ det01 xgap 0.0                        # Gap between the square detector pixels (
 det01 ygap 0.0                        # Gap between the square detector pixels (expressed as a fraction of the y pixel size -- x is predominantly the dispersion axis)
 det01 ysize 1.0                       # The size of a pixel in the y-direction as a multiple of the x pixel size (i.e. xsize = 1.0 -- x is predominantly the dispersion axis)
 det01 darkcurr 0.0                    # Dark current (e-/hour)
-det01 ronoise 4.64                    # Average of values on 4 detectors; Read-out noise (e-)
-det01 gain 1.0                        # Gain (e-/ADU) values input in arlris.py (instead of arproc.py where they are usually read in)
 det01 saturation 65535.0              # The detector Saturation level
 det01 nonlinear 0.76                  # Percentage of detector range which is linear (i.e. everything above nonlinear*saturation will be flagged as saturated)
-## PULL THE FOLLOWING FROM THE HEADER WHEN POSSIBLE (FOR BINNING ESPECIALLY)##
 det01 numamplifiers 2                 # Number of amplifiers
+det01 gain 1.255,1.18                 # Gain (e-/ADU) values for the 2 amplifers
```

Comment:
I am pretty sure it is backwards compatible.
But we do want to update LRISb.  
Doing so now..

> On May 1, 2016, at 7:32 PM, Ryan Cooke <notifications@github.com> wrote:
> 
> In src/settings.lris_red <https://github.com/PYPIT/PYPIT/pull/60#discussion_r61698819>:
> 
> >  det01 numamplifiers 2                 # Number of amplifiers
> > +det01 gain 1.255,1.18                 # Gain (e-/ADU) values for the 2 amplifers
> We also need to update the LRISb and KastB settings files to match the new code, right?
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub <https://github.com/PYPIT/PYPIT/pull/60/files/a0e6a39f845c44a29a1def02df472dea5469cb7d#r61698819>
> !DSPAM:826,5726bc4b1812697122454!

----------------------------------------------
J. Xavier Prochaska
UCO/Lick Observatory
1156 High St.
UC Santa Cruz
Santa Cruz, CA 95064

http://www.ucolick.org/~xavier/
http://imps.ucolick.org/
http://www.qpqsurvey.org/
http://k1dm3.ucolick.org/
http://www.ucolick.org/~xavier/IDL/index.html
https://github.com/pyigm/pyigm
https://github.com/linetools/linetools

831-459-2135 (Direct)
831-459-2991 (UCO/Lick Main)
831-459-5244 (Fax)